cfut/remote.py

```diff
@@ -10,9 +10,16 @@ def format_remote_exc():
     tb = tb.tb_next  # Remove root call to worker().
     return ''.join(traceback.format_exception(typ, value, tb))
 
-def worker(workerid):
+def worker(workerid, extra_import_paths="!"):
     """Called to execute a job on a remote host."""
     print("worker")
+    if extra_import_paths != '!':
+        extra_import_paths = extra_import_paths.split(':')
+        print("Prepending %d paths to sys.path:" % len(extra_import_paths))
```

Comment:
Just to point out my own slowness, I _almost_ commented here that this output would look weird if there are no paths to add, but of course this is all under a `!= '!'` check.